keycloak/themes/uid2-theme/email/html/template.ftl

```diff
@@ -0,0 +1,16 @@
+<#macro emailLayout>
+<html>
+<body>
+  <div style="padding: 70px; text-align: center; font-family: 'Inter', sans-serif">
+    <div style="padding: 20px">
+      <a href="http://localhost:3000/"
+        ><img src="http://localhost:3000/uid2-logo.png" alt="UID2 logo" style="height: 30px"
```

Comment:
how would this get handled in prod ? would this need to be replaced with a cdn url ?

Reply:
Yes, this will be updated once we have a cdn for producation

Reply:
@jingyi-gao-ttd can you add a ticket to Jira to make sure it gets done?

Reply:
@lionell-pack-ttd Here is the ticket https://atlassian.thetradedesk.com/jira/browse/UID2-985